src/clj/game/core/ice.clj

```diff
@@ -242,7 +242,7 @@
 
 (defn resolve-subroutine!
   ([state side ice sub]
-   (let [eid (make-eid state {:source (:title ice)
+   (let [eid (make-eid state {:source ice
```

Comment:
lol thanks for this, it was a dumb decision back in [2018](https://github.com/mtgred/netrunner/commit/34be048e63b0f1975256bd168cd672991d2ab602) and I'm surprised no one has fixed it yet.